articles/cognitive-services/text-analytics/whats-new.md

```diff
@@ -16,7 +16,20 @@ ms.author: aahi
 
 The Text Analytics API is updated on an ongoing basis. To stay up-to-date with recent developments, this article provides you with information about new releases and features.
 
-## Named Entity Recognition v3 public preview - February 2020
+## February 2020
+
+### SDK support for Text Analytics API v3 Public Preview
+
+As part of the January 2020 unified Azure SDK [release](https://techcommunity.microsoft.com/t5/azure-sdk/january-2020-unified-azure-sdk-release/ba-p/1097290), Text Analytics API v3 SDK is now available as a public preview for the following programming languages:
```

Comment:
I feel like we can remove the January 2020 mention, since it feels a bit confusing under the "February" heading.

```suggestion
As part of the [unified Azure SDK release](https://techcommunity.microsoft.com/t5/azure-sdk/january-2020-unified-azure-sdk-release/ba-p/1097290), the Text Analytics API v3 SDK is now available as a public preview for the following programming languages:
```